qemu/tests/qemu_img_convert_from_vdi.py

```diff
@@ -0,0 +1,41 @@
+from avocado import fail_on
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+from provider import qemu_img_utils as img_utils
+
+
+def run(test, params, env):
+    """
+    1. Convert to a target image
+    2. Boot up a guest from the target file
+    3. Check the md5 value of the temporary file, and make sure the values are
+    the same
```

Comment:
Please update the doc string, not found check md5 value related codes.
e.g.
1. Convert from vdi os image
2. Boot up from the target image
3. Check the data integrity

Reply:
No need this step, and updated. Thanks.